test/CMakeLists.txt

```diff
@@ -1176,6 +1187,42 @@ if(iodaconv_bufr_ENABLED)
                             gdas.t06z.adpsfc_snow.tm00.nc ${IODA_CONV_COMP_TOL_ZERO}
                     DEPENDS bufr2ioda.x )
 
+  ecbuild_add_test( TARGET  test_iodaconv_bufr_ncep_aircar2ioda
```

Comment:
There's one for each yaml
1. aircar bufr
2. aircft amdar bufr
3. aircft no amdar bufr
4. aircft prepbufr (acft_profiles)

Reply:
Up to you all if you want to keep them. There are slight differences that I think are important but I don't know enough about other non-aircraft yamls to know if they're important to keep in.

Reply:
At the unit level testing it's more about code coverage. So if you think that having the four tests will hit more of the "if" branches in your code, then it might be worth keeping all these. The tradeoff of course is that we want the tier1 ctests to run as fast as possible so tiny test cases are best.

Reply:
The tests take a fraction of a second each. I think we should keep them, as they also test different portions of aircraftAltitude.